.changeset/famous-olives-shake.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Added changeset to test out logic.

The failing scenario was tested in [this run](Tested out failing scenario in [this run](https://github.com/FuelLabs/fuels-ts/actions/runs/8836716368/job/24263942928?pr=2166#step:6:20) with [this commit](https://github.com/FuelLabs/fuels-ts/pull/2166/commits/c233f131761e7344f6dc26e3765282c4c32fde7b#diff-3dc21aee46e1f7f3caa87f858f39062ed869ae19925689241ae4c37f19b31e54R2).